src/lib/targeting.ts

```diff
@@ -29,8 +29,6 @@ export interface ThrottleConfig {
     minDaysBetweenViews: number;
 }
 
-const defaultThrottle = { maxViewsDays: 90, maxViewsCount: 4, minDaysBetweenViews: 5 };
```

Comment:
This object wasn't in use

Reply:
Thanks - meant to remove that in my last PR and forgot :(